tests/unit/utils/bind-data-test-attributes-test.js

```diff
@@ -119,18 +120,29 @@ test('it skips if attributeBindings is a computed property', function(assert) {
   assert.deepEqual(instance.get('attributeBindings'), ['foo']);
 });
 
-test('it skips if tagName is empty', function(assert) {
-  let Fixture = EmberObject.extend({
-    tagName: '',
-    'data-test-from-factory': 'foo',
+if (!config.stripTestSelectors) {
```

Comment:
I don't understand this. Why are you checking `config.stripTestSelectors` here? The `bindDataTestAttributes()` function doesn't seem to care about that config option 🤔